### ISSUE TITLE
Generation quality of the nuimages-model

### ISSUE THREAD

**Issue by zhangxiao696** [KaiChen1998/GeoDiffusion#11]
hello,
   when I follow your codes train nuimages dataset, the clarity of image generation is very low
![n008-2018-09-12-09-06-08-0400__CAM_BACK_LEFT__1536758322547405_2](https://github.com/KaiChen1998/GeoDiffusion/assets/13326803/248e4405-444f-413b-bbb8-baff9e211969)

![n008-2018-09-14-14-02-31-0400__CAM_BACK__1536948703937558_4](https://github.com/KaiChen1998/GeoDiffusion/assets/13326803/698c8edd-3819-448c-8bee-3d696ff58e92)

**Comment by KaiChen1998** [comment]
@zhangxiao696 Actually, I would say these demo images demonstrate quite good quality :), considering the default setting we provide is based on `256x256` resolution, which is significantly lower than the original `1600x900` real images. 

I do recommend you increase the image resolution to `512x512` or even larger. What you need to do is simply modify the image resolution in the [data config file](https://github.com/KaiChen1998/GeoDiffusion/blob/main/configs/data/nuimage_256x256.py#L8) and scale your learning rate linearly with you batch size. Consider adopting the gradient accumulation via the `--gradient_accumulation_steps` argument if necessary for your GPU resources.
